common/src/main/java/tachyon/util/FormatUtils.java

```diff
@@ -41,19 +47,39 @@ public static String parametersToString(Object... objs) {
     return sb.toString();
   }
 
-  public static void printByteBuffer(Logger LOG, ByteBuffer buf) {
+  /**
+   * Print the given buffer.
+   *
+   * @param logger logger to use
+   * @param buf buffer to use
+   */
+  public static void printByteBuffer(Logger logger, ByteBuffer buf) {
     StringBuilder sb = new StringBuilder();
     for (int k = 0; k < buf.limit() / 4; k ++) {
```

Comment:
I think all of the buffer will be printed. The function prints the content of the buffer in 4-byte increments as it uses getInt() to fetch the bytes. I have expanded a comment to explain this.